packit_service_centosmsg/consumer.py

```diff
@@ -62,11 +61,14 @@ def celery_app(self):
     def on_message(self, client, userdata, msg):
         pprint(msg.topic)
         pprint(json.loads(msg.payload))
-        # self.celery_app.send_task(
-            # name="task.steve_jobs.process_message", kwargs={"event": msg}
-        # )
-        logger.info(json.loads(msg.payload))
 
+        if os.getenv("SEND_CELERY_TASKS"):
```

Comment:
Why do we want it configurable?

Reply:
I would like to have the ability to turn this on and off in OpenShift while integrating it with the current setup, without the need to rebuild the container image.

For example if sending these tasks causes trouble, would be nice to turn it off by setting an env var.

Reply:
OK, that makes sense.

Reply:
I already said it in my previous review, but maybe it got overlooked ;-)
What is the more common scenario, sending them or not sending them?
I mean, if you only want to be able to turn the sending off while integrating this, then I'd suggest something like `DISABLE_SENDING_CELERY_TASKS`, because `SEND_CELERY_TASKS` suggests that we don't send them by default.

Reply:
Okay, this makes sense. I'll update to use `DISABLE_SENDING_CELERY_TASKS`.